README.md

```diff
@@ -25,7 +25,7 @@ If you have problems syncing to the right chain, you can force your client to co
 
 For geth, start the client without connecting to any peers:
 ```
-geth --testnet --fast --nodiscover
+geth --testnet --fast --nodiscover --rpc --rpcport "8545" --rpcapi "admin,db,eth,net,web3,personal"
```

Comment:
isn't that the default setting?

Reply:
no, if you not type those parameters,testnet can't get 8545 port work find.

Reply:
I create another PR to update the Document